internal/config/cgmgr/cgroupfs.go

```diff
@@ -38,7 +38,7 @@ func (*CgroupfsManager) ContainerCgroupPath(sbParent, containerID string) string
 	if sbParent != "" {
 		parent = sbParent
 	}
-	return filepath.Join(parent, crioPrefix+"-"+containerID)
+	return filepath.Join("/", parent, crioPrefix+"-"+containerID)
```

Comment:
IDK, but should we be doing a securejoin here?

Reply:
I don't think so.

1. This is not an absolute path, but a one relative to /sys/fs/cgroup.
2. runc/crun [should] do sanity checks.
3. The only untrusted component here is parent, this is a cgroup of sandbox, which is also created by cri-o.